ID: 12776
Title: Can't import database
Description:
### What happens?

After exporting a database made with an older version of DuckDB (likely originally 0.9.x), it cannot be imported. The error I get is:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Java\Source\Idea\mallard\mallard\shrink_db.py", line 28, in <module>
    con.execute("import database '/data/mallard/mallard_shrunk';")
duckdb.duckdb.CatalogException: Catalog Error: Schema with name "information_schema" already exists!
```

I'm wondering if it's related to this issue: https://github.com/duckdb/duckdb/pull/12456

The top of schema.sql in the export directory reads:

```
CREATE SCHEMA information_schema;
CREATE SCHEMA pg_catalog;
```

Note: the original databases I used in my tests were probably created originally with a 0.9 version, maybe 0.8 but I doubt it. If I create a new DB and table with 1.0 CLI, the `CREATE SCHEMA` statements aren't in schema.sql. But I suspect a lot of files will be affected 0.9 wasn't so long ago.

### To Reproduce

Create a database with an older version (0.9), and then export it with:

EXPORT DATABASE 'dir_name';

Then try to import it with:

IMPORT DATABASE 'dir_name'

I was following the workaround given in this issue to reclaim space:

https://github.com/duckdb/duckdb/issues/8261#issuecomment-1911141918

I followed that code using 0.10.0. I then installed the 1.0 CLI and reproduced it with a second database.

### OS:

Windows 11 Pro x64

### DuckDB Version:

0.10.0 and 1.0

### DuckDB Client:

Python 3.11 and CLI

### Full Name:

Philip Yurchuk

### Affiliation:

Made Up Name

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

No - Other reason (please specify in the issue body)

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have